beta/plugins/markdownToHtml.mjs

```diff
@@ -6,18 +14,16 @@ const unrwapImages = require('remark-unwrap-images'); // Removes <p> wrapper aro
 const smartyPants = require('./remark-smartypants'); // Cleans up typography
 const html = require('remark-html');
 
-module.exports = {
-  remarkPlugins: [
-    externalLinks,
-    customHeaders,
-    images,
-    unrwapImages,
-    smartyPants,
-  ],
-  markdownToHtml,
-};
+export const remarkPlugins = [
+  remarkGfm,
+  externalLinks,
+  customHeaders,
```

Comment:
This plugin is a really ancient copy/paste version — i don‘t think it’ll work anymore.
I’d recommend using https://github.com/rehypejs/rehype-autolink-headings instead, I think it’s a modern replacement.

I’d also recommend replacing `remark-external-links` with `rehype-external-links`.

Note that both plugins are called `rehype-*`. That’s because they work on an HTML ast.
`remark` is the markdown ecosystem — `rehype` is the HTML ecosystem.